src/cli/providers/messenger/attachment.js

```diff
@@ -26,6 +26,7 @@ const help = () => {
     ${chalk.dim('Options:')}
 
       --force   Upload all assets and regenerate bottender-lock.json.
+      --yes     Skip prompt confirmation for force upload all assets.
```

Comment:
Bottender CLI will not force upload when not passing `--force` option, so the description may be misunderstood.